### PR TITLE
docs: translate Queueing a Series of State Updates page

### PR DESCRIPTION
**Pull request by milhamm**
Closes https://github.com/reactjs/id.react.dev/issues/449

## Description

Translate the Queueing a Series of State Updates page.
Page URL: https://id.react.dev/learn/queueing-a-series-of-state-updates

### CODE REVIEW

**Comment by mhaidarhanif** [comment]
@milhamm I thought it was fine because @mrevanzak hasn't done any PR.

File: src/content/learn/queueing-a-series-of-state-updates.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
 ---
-title: Queueing a Series of State Updates
+title: Mengantre Serangkaian Pembaruan State
 ---
 
 <Intro>
 
-Setting a state variable will queue another render. But sometimes you might want to perform multiple operations on the value before queueing the next render. To do this, it helps to understand how React batches state updates.
+Mengatur variabel *state* akan menambahkan antrean *render* baru. Terkadang Anda ingin melakukan beberapa operasi terhadap nilai sebelum menambahkan antrean *render* selanjutnya. Untuk melakukanya, penting untuk memahami bagaimana React melakukan pembaruan *state* secara berkelompok.
```
Comment (mhaidarhanif) [inline src/content/learn/queueing-a-series-of-state-updates.md:7]: Introduce a new word along with the original English word. So we have a better context.

```suggestion
Mengatur variabel *state* akan menambahkan antrean (*queue*) *render* baru. Terkadang Anda ingin melakukan beberapa operasi terhadap nilai sebelum menambahkan antrean *render* selanjutnya. Untuk melakukanya, penting untuk memahami bagaimana React melakukan pembaruan *state* secara berkelompok. 
```

**Review by mhaidarhanif — CHANGES_REQUESTED** [review]
Suggestion

File: src/content/learn/queueing-a-series-of-state-updates.md (hunk 1 above)
Comment (milhamm) [reply src/content/learn/queueing-a-series-of-state-updates.md:7]: Sure, thanks for the early review

Comment (mhaidarhanif) [reply src/content/learn/queueing-a-series-of-state-updates.md:7]: @milhamm Sure thing, you're welcome

**Comment by milhamm** [comment]
Handed over to @mrevanzak

**Pull request closed without merging**